crates/bevy_app/src/plugin.rs

```diff
@@ -1,8 +1,41 @@
 use downcast_rs::{impl_downcast, Downcast};
 
-use crate::App;
+use crate::{App, InternedAppLabel};
 use std::any::Any;
 
+/// Plugin state in the application
```

Comment:
More docs please! Information about how to access this data in the form of a doc test would be particularly valuable.